daprdocs/content/en/concepts/components-concept.md

```diff
@@ -6,21 +6,48 @@ weight: 300
 
- You can get a list of current components available in the hosting environment using the `dapr components` CLI command.
+- The [components-contrib repository](https://github.com/dapr/components-contrib)
+- [Pluggable components]({{<ref "components-concept.md#pluggable-components" >}}).
+
+A building block can use any combination of components. For example, the [actors]({{<ref "actors-overview.md">}}) and the [state management]({{<ref "state-management-overview.md">}}) building blocks both use [state components](https://github.com/dapr/components-contrib/tree/master/state).
+
+As another example, the [pub/sub]({{<ref "pubsub-overview.md">}}) building block uses [pub/sub components](https://github.com/dapr/components-contrib/tree/master/pubsub).
+
+You can get a list of current components available in the hosting environment using the `dapr components` CLI command.
 
 ## Component specification
 
-Each component has a specification (or spec) that it conforms to. Components are configured at design-time with a YAML file which is stored in either a `components/local` folder within your solution, or globally in the `.dapr` folder created when invoking `dapr init`. These YAML files adhere to the generic [Dapr component schema]({{<ref "component-schema.md">}}), but each is specific to the component specification. 
+Each component has a specification (or spec) that it conforms to. Components are configured at design-time with a YAML file which is stored in either:
+
+- A `components/local` folder within your solution, or
+- Globally in the `.dapr` folder created when invoking `dapr init`.
+
+These YAML files adhere to the generic [Dapr component schema]({{<ref "component-schema.md">}}), but each is specific to the component specification.
 
 It is important to understand that the component spec values, particularly the spec `metadata`, can change between components of the same component type, for example between different state stores, and that some design-time spec values can be overridden at runtime when making requests to a component's API. As a result, it is strongly recommended to review a [component's specs]({{<ref "components-reference">}}), paying particular attention to the sample payloads for requests to set the metadata used to interact with the component.
 
 The diagram below shows some examples of the components for each component type
 <img src="/images/concepts-components.png" width=1200>
 
+## Built-in and pluggable components
+
+Dapr has built-in components that are included as part of the runtime. These are public components that are developed and donated by the community and are available to use in every release. 
+
+Dapr also allows for users to create their own private components called pluggable components. These are components that are self-hosted (exe or container), do not need to be written in Go, exist outside the Dapr runtime, and are able to "plug" into Dapr to utilize the building block APIs.
```

Comment:
@msfussell @hhunter-ms, can we change from `exe` to `process`?

I mean, .exe is just a windows file extension for executable

```suggestion
Dapr also allows for users to create their own private components called pluggable components. These are components that are self-hosted (binary or container), do not need to be written in Go, exist outside the Dapr runtime, and are able to "plug" into Dapr to utilize the building block APIs.
```

Reply:
already fixed